### PR TITLE
add fund dist feature to the network launching pipeline

### PR DESCRIPTION
**Pull request by AndyBoWu**
note that 
* key.sh is the file to store the confidential bls key 
* this file has been created in the devops instance
* required env is ready on devops

### CODE REVIEW

File: pipeline/go.sh [hunk 1]
```diff
@@ -24,6 +28,7 @@ This script automates the benchmark test based on profile.
    -k                keep all the instances, skip deinit (default: $KEEP)
    -t                do not run txgen (default: $TXGEN), overriding profile configuration
    -w                run wallet test (default: $WALLET)
+   -n                chain_id (eg., testnet)
```
Comment (LeoHChen) [inline pipeline/go.sh]: no need to add this, the info can be found in benchmark-profile.json file.

File: pipeline/go.sh [hunk 2]
```diff
@@ -572,6 +580,22 @@ function do_replace
...
+# do fund distribution 
+function do_fund_dist
+{
+   echo "start to distribute funds to test accounts.. " 
+   local timeout amount
+   timeout=120
+   amount=100000
+   if [ "$CHAIN_ID" == 'testnet' ]; then
+      endpoints=("https://api.s0.os.hmny.io/" "https://api.s1.os.hmny.io/" "https://api.s2.os.hmny.io/")
+   elif [ "$CHAIN_ID" == 'lrtn' ]; then
```
Comment (LeoHChen) [inline pipeline/go.sh]: the configuration can be found in benchmark-<profile>.json file, check 'rpczone' , and 'network_type'.

**Comment by AndyBoWu** [comment]
@LeoHChen dont merge this feature just yet, I am adding a feature to retrieve explorer IP by
* remove the network information
* deduce information for benchmark-*.json file

**Comment by AndyBoWu** [comment]
will automat this job on jenkins

**Pull request closed without merging**